test/e2e/performanceprofile/functests/utils/cgroup/config/config.go

```diff
@@ -0,0 +1,20 @@
+package config
+
+import (
+	"context"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+const CgroupMountPoint = "/sys/fs/cgroup"
+
+type Config struct {
+	CPUSet string
+	Quota  string
+	Period string
+}
+
+type Getter interface {
```

Comment:
I'd actually get rid of this interface entirely.
Plus, in golang better to define interface on the consumer side: https://www.thoughtworks.com/insights/blog/programming-languages/mistakes-to-avoid-when-coming-from-an-object-oriented-language

Reply:
Thank you for the useful information, it really helps understand the principles better. 

So in our case it means the interface would be implemented in the test files themselves, i.e. `7_performance_kubelet_node/cgroups.go` or `11_mixedcpus/mixedcpus.go` . 
But what if both packages want to use the exact interface? should both declare it?

And my second questions is how do you suggest the `BuildGetter` to return a struct, while it suppose to return an abstraction for getting the cgroup information.
 
Of course we can implement the `BuildGetter` on the consumer side, but again it's a duplication of code, since at least two packages (from what I know of) would need to implement it to retrieve the existing cgroup configuration on the node.